spring-kafka/src/main/java/org/springframework/kafka/core/StreamsBuilderFactoryBean.java

```diff
@@ -52,15 +58,19 @@
 
 	private KafkaStreams kafkaStreams;
 
+	private KafkaStreamsCustomizer kafkaStreamsCustomizer = new CompositeKafkaStreamsCustomizer();
```

Comment:
This instance for nothing doesn't have too much value.
Only what it safes for you is just simple `if (this.kafkaStreamsCustomizer != null)`.
I'd prefer such an `if`, instead of extra method call.